README.md

```diff
@@ -64,74 +87,8 @@ module EmberComponentHelper
 end
 ```
 
-And then use that inside of your view files instead.
+Then you can use that helper in your view files.
 
 ```html+erb
 <%= ember_component 'my-component', title: @post.title, body: @post.body %>
 ```
-
-## Bypassing the Addon
```

Comment:
I think it's still worth having this section, even if it's just the example you had in #14